scripts/execution.py

```diff
@@ -78,18 +77,25 @@ def run(self, args: Sequence[str], cwd: str = None,
                               ex.returncode)
             raise
 
+        if self.verbose:
+            sys.stdout.write(result.stdout)
```

Comment:
This will have a side-effect: stdout and stderr will now be printed separately from each making the logs harder to interpret. Pease write a separate method for running a process with stdout and stderr capture.